index.js

```diff
@@ -46,7 +49,10 @@ console.log('hello world')
  * Example Output: "$49.90"
  */
 
-// TODO: ADD CODE HERE
+let price = 9.98;
+let quantity = 5;
+let total = price * quantity;
+console.log (Math.round(total));
```

Comment:
good idea to use Math here. this will actually though just round to a whole number, so you should try to use the toFixed () method to round to 2 decimal places. Feel free to start another branch off master if you'd like to make additional changes, then you can merge them in to master just like you did with this PR.